lib/internal/freeze_intrinsics.js

```diff
@@ -378,10 +422,10 @@ module.exports = function() {
             `Cannot assign to read only property '${prop}' of object '${obj}'`
           );
         }
-        if (objectHasOwnProperty.call(this, prop)) {
+        if (FunctionPrototypeCall(objectHasOwnProperty, this, prop)) {
```

Comment:
This should use `ObjectPrototypeHasOwnProperty` directly from `primordials`:
```suggestion
        if (ObjectPrototypeHasOwnProperty(this, prop)) {
```